questions/question.css

```diff
@@ -0,0 +1,83 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;600&display=swap');
+
+
+* {
+    box-sizing: border-box;
+    color: white;
+    font-size: 1.2rem;
+}
+
+body {
+    margin: 0;
+    width: 100%;
+    font-family: 'Montserrat', sans-serif;
+    background-color: rgb(6, 6, 49);
+}
+
+ul {
+    list-style: none;
+}
+h1 {
+    font-size: 1.8rem;
+}
+main {
+    width: 600px;
+    text-align: center;
+    margin: auto;
+    
+}
+
+.score {
+    display: flex;
+    justify-content: space-between;
+}
+.timmer {
+    position: fixed;
+    top: 0;
+    right: 1rem;
+}
+
+/* .button{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+} */
+.button {
+    margin-top: 2.5rem;
```

Comment:
Can you global variables for properties like margin, padding and gap

Reply:
Yes Prabhav I will do that